pkgs/tools/misc/rmlint/default.nix

```
@@ -36,6 +37,11 @@ stdenv.mkDerivation rec {
   patches = [
     # pass through NIX_* environment variables to scons.
     ./scons-nix-env.patch
+    # fix #664
```

```suggestion
    # fix https://github.com/sahib/rmlint/issues/664
```

Comment `#664` alone doesn't provide enough context to understand what it means.